less/pages/blog.less

```diff
@@ -14,7 +14,7 @@
 main > article {
   // limit content width to increase readability and center it
   // only affect those on the section pages, not the ones on the front page
-  max-width: 60rem;
+  max-width: 45rem;
```

Comment:
I'm not sure this helps. 60rem is already not all that wide (i.e. fairly well readable IMHO), and code samples will be cut off at 45rem.

Reply:
The widely (no pun intended) adopted max width for readable screen prose (and forms) for wide responsive layouts at a base (i.e., 1rem) font size of 16px (which is the default for most browsers) is 40-50rem. This is to avoid exceeding ~100 characters per line (cpl), which is already a considerable increase from the print standard of 45-75 (ideally 66) cpl.

Simply reducing the max width without making complementary adjustments[^adjustments] indeed makes the site look a bit sparse—and we’ve discussed defining a design system to address such concerns—but this change (along with everything else is this PR) was made as a bare minimum to adhere to best practices in the meantime.

A note regarding code samples: while it’s true that this commit as-is would limit fenced code display to 45rem, it could be adjusted such that, e.g., the fenced code (i.e., the content itself as opposed to its container) is aligned with the prose, thereby returning the padding width otherwise lost by the offset. For that matter, the styled containers could stretch to the full page width, etc., but again, these are decisions best made within a design system.

[^adjustments]: E.g., slightly increasing font size and leading, applying a balanced color palette, introducing additional design elements to help guide the eye (also enabling asymmetrical layouts as are practically ubiquitous among major online outlets of prose), etc.

Reply:
Alright. Thanks for the elaboration. I'm still "meh" about this for the exact reasons you outlined, but I won't stall the change because of that.